src/pfe/file-watcher/server/test/functional-test/suites/project/tests/project-specification.ts

```diff
@@ -322,6 +325,61 @@ export function projectSpecificationTest(socket: SocketIO, projData: projectsCon
             }
         }).timeout(timeoutConfigs.defaultTimeout);
 
+        it(`set internalPort to ${constants.disablePingPort} in .cw-settings file to check for serverless app`, async () => {
+            let events, eventDatas;
+
+            events = [eventConfigs.events.settingsChanged, eventConfigs.events.statusChanged];
+            eventDatas = [{
+                "projectID": projData.projectID,
+                "status": "success",
+                "ports": {
+                    "internalPort": `${constants.disablePingPort}`,
+                }
+            }, {"projectID": projData.projectID, "appStatus": "stopped"}];
+            await reconfigProjectSettings("internalPort", constants.disablePingPort, events, eventDatas);
```

Comment:
do we need to add `.timeout(timeoutConfigs.defaultTimeout);` on the func call like it is done below?

Reply:
The function call does not require timeout. It will all fall under one timeout for the `it` case which is set by `.timeout(timeoutConfigs.defaultTimeout);`